spec/unit/rendezvous/rendezvous.spec.ts

```diff
@@ -278,7 +278,7 @@ describe("Rendezvous", function () {
         expect(aliceOnFailure).toHaveBeenCalledWith(RendezvousFailureReason.UnsupportedAlgorithm);
     });
 
-    it("new device declines protocol", async function () {
+    it("new device declines protocol (bad protocol)", async function () {
```

Comment:
@hughns Based on https://github.com/matrix-org/matrix-js-sdk/pull/2747, do these test names look accurate?

I've renamed them because their titles were duplicated before which doesn't comply with the new `jest/no-identical-title`

The tests are so similar

Reply:
I've tweaked the names to better match.

I have made a note to refactor the common code from the similar cases to make them more readable. 👍

Reply:
Thanks for updating it directly @hughns 👍